test/models/user_test.rb

```diff
@@ -26,6 +26,12 @@ def setup
     refute User.new(email: test_email).valid?
   end
 
+  test 'email must be saved downcased' do
+    new_email = 'NEW_EMAIL@exaMple.cOm'
+    assert create(:user, email: new_email.downcase)
+    refute User.new(email: new_email).valid?
```

Comment:
@rickr I tried your suggestion to try to get my test to fail and it wouldn't recognize the test at all...it seemed to pass without the `before_save`...so I looked into the config/initializers/devise.rb line 49 downcases emails upon user creation. If I'm understanding this right I'm having trouble testing the `before_save` because devise is already downcasing the emails. Is that right?